src/document-info.js

```diff
@@ -56,11 +56,16 @@ export function documentInfoForDoc(nodeOrDoc) {
           canonicalUrl = parseUrl(canonicalTag.href).href;
         }
         const pageViewId = getPageViewId(ampdoc.win);
-        return {url, sourceUrl, canonicalUrl, pageViewId};
+        const res = {sourceUrl, canonicalUrl, pageViewId};
+        Object.defineProperty(res, 'sourceUrl', {
```

Comment:
It's necessary, because of [this comment](https://github.com/ampproject/amphtml/pull/5606#pullrequestreview-4780791)